crates/ty_python_semantic/src/types/builder.rs

```diff
@@ -374,6 +375,44 @@ impl<'db> UnionBuilder<'db> {
                     self.elements.swap_remove(index);
                 }
             }
+            Type::EnumLiteral(enum_literal) => {
```

Comment:
nit: there's a lot of literals floating around in this branch. Something like this name might be a bit clearer?

```suggestion
            Type::EnumLiteral(enum_member_to_add) => {
```